connector/connect/common/src/test/resources/query-tests/explain-results/function_aes_decrypt.explain

```diff
@@ -1,2 +1,2 @@
-Project [staticinvoke(class org.apache.spark.sql.catalyst.expressions.ExpressionImplUtils, BinaryType, aesDecrypt, cast(g#0 as binary), cast(g#0 as binary), GCM, DEFAULT, cast( as binary), BinaryType, BinaryType, StringTypeAnyCollation, StringTypeAnyCollation, BinaryType, true, true, true) AS aes_decrypt(g, g, GCM, DEFAULT, )#0]
```

Comment:
I don't have a strong preference but I feel like we should show full name of `org.apache.spark.sql.catalyst.expressions.ExpressionImplUtils`.

Reply:
Hi @HyukjinKwon,

I understand and share your concern. Unlike the symbolized and maybe well-known TreeNode names, the arbitrary class names are not that recognizable if we use abbreviations instead of full names here.

The use of full or abbreviated names can be summarized as follows: 1) presenting a Spark plan, and 2) locating the call site when errors occur. 

In the former case, the full class names always share the same prefix `org.apache.spark.sql.catalyst.expressions` or are provided by users, which could be considered redundant. In the latter case, searching for abbreviated names in modern IDEs does not necessarily have any significant disadvantages compared to full names.

Therefore, using shorter ones can prettify the plans without losing any benefits. WDYT?

Reply:
Spark only uses a few fixed classes in StaticInvoke/Invoke, so short names will be fine. If third-party Spark plugins also create StaticInvoke/Invoke, then we may have ambiguity.

One idea is to use short package names if the classes do live in Spark, e.g. `catalyst.expressions.ExpressionImplUtils`

Reply:
The third-party plugins sometimes also use `org.apache.spark.*` just like some of our packages share the same scope of `hive`, `parquet` etc, so it becomes tricky to determine whether classes live in spark jars.

Maybe we can apply to all classes instead of spark classes only as A %c[3] pattern class name with a function name could reduce ambiguity to a necessary degree for these log-wise outputs

Reply:
OK, we don't need to care about internal or external usages, but just always ignore the `org.apache.spark` prefix.

Reply:
I found this exception - org.apache.spark.unsafe.types.ByteArray

Reply:
I mean, we can omit all these common spark packages: `org.apache.spark.sql.catalyst`, `org.apache.spark.sql`, `org.apache.spark`, etc.. This shouldn't introduce ambiguity unless people want to sabotage: use the same class name but a different spark package.

Reply:
Do you mean we can use simple names for `org.apache.spark.*` and full names for others

Reply:
Yes!